index.md

```diff
@@ -6,8 +6,9 @@ github:
   repository: w3c/wai-intro-act
 feedbackmail: wai@w3.org
 footer: >
-  <p><strong>Date:</strong> Updated 30 October 2020.</p>
-  <p><strong>Editors:</strong> <a href="https://www.w3.org/People/shadi/">Shadi Abou-Zahra</a> and <a href="http://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
```

Comment:
I'm going to leave Shadi as lead editor because he was lead editor on the content that is there. If we change it substantively later, we can move Shadi to previous editor.